### PR TITLE
Update config.yaml to support /reclears and /namecolor for NAUR

### PR DESCRIPTION
**Pull request by OTCompa**
Won't impact anything until #25 and #26 are merged. Just adds the necessary things to enable the two commands in NAUR. Original PR here: https://github.com/brtran4/clearingway/pull/2

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -618,6 +644,27 @@ guilds:
       - name: "TOP Cleared"
         type: "Cleared"
         color: 0x6534c6
+      - name: "Reclear TOP"
+        type: "Reclear"
+        color: 0x6534c6
+      - name: "The Alpha Color"
+        type: "Name Color"
+        color: 0x6534c6
+  # - ids: []
```
Comment (ghost) [inline config.yaml:654]: Prepared for FRU release, can be uncommented when the fight goes live.

Comment (brtran4) [reply config.yaml:654]: just like the very first PR of the chain, let's add a `// TODO: ... ` comment

**Review by brtran4 — CHANGES_REQUESTED** [review]
we should make the naming convention consistent. if there is `TOP Cleared` we should likewise name the reclear role as `TOP Reclear` or vice versa: `Cleared TOP` and `Reclear TOP`. we shouldn't break the naming convention here and make it as consistent as possible

**Comment by ghost** [comment]
> we should make the naming convention consistent. if there is `TOP Cleared` we should likewise name the reclear role as `TOP Reclear` or vice versa: `Cleared TOP` and `Reclear TOP`. we shouldn't break the naming convention here and make it as consistent as possible

disagree with this because it makes it easier for the user to start typing Reclear and see the options. If anything we should reverse the other roles but thats not our decision

**Comment by brtran4** [comment]
> > we should make the naming convention consistent. if there is `TOP Cleared` we should likewise name the reclear role as `TOP Reclear` or vice versa: `Cleared TOP` and `Reclear TOP`. we shouldn't break the naming convention here and make it as consistent as possible
> 
> disagree with this because it makes it easier for the user to start typing Reclear and see the options. If anything we should reverse the other roles but thats not our decision

the `Cleared` roles should not be auto-populating as a parameter

**Comment by ghost** [comment]
> > > we should make the naming convention consistent. if there is `TOP Cleared` we should likewise name the reclear role as `TOP Reclear` or vice versa: `Cleared TOP` and `Reclear TOP`. we shouldn't break the naming convention here and make it as consistent as possible
> > 
> > 
> > disagree with this because it makes it easier for the user to start typing Reclear and see the options. If anything we should reverse the other roles but thats not our decision
> 
> the `Cleared` roles should not be auto-populating as a parameter

im talking about Relcear

image

image

you can see the difference when someone is typing out the role. it wont show relcears as easily if the ultimate is typed first

**Comment by ghost** [comment]
but it doesnt matter i guess

**Comment by brtran4** [comment]
i'm still in favor of consistency especially since there's also a prog role that also follows the same naming principle

**Comment by ghost** [comment]
@OTCompa 
image
just change the roles to this so its consistent as Rem said

**Review by ghost — APPROVED** [review]

**Review by brtran4 — CHANGES_REQUESTED** [review]
still needs that quick todo comment

**Review by brtran4 — APPROVED** [review]

**Pull request merged**